netlify.toml

```diff
@@ -5,10 +5,10 @@ command = "yarn build --environment=staging"
 command = "yarn build --environment=production"
 
 [context.develop]
-ZAP_API_STAGING_URL = "https://zap-api-staging.herokuapp.com"
+ZAP_API_URL = "https://zap-api-staging.herokuapp.com"
```

Comment:
Ok I think I actually told you the wrong syntax here. Based on what we have in [factfinder's `netlify.toml`](https://github.com/NYCPlanning/labs-factfinder/blob/develop/netlify.toml), it looks like it expects that `environment = {}` syntax

Reply:
@TylerMatteo  `netlify.toml` updated.